modules/olmv1-installing-an-operator.adoc

```diff
@@ -7,40 +7,115 @@
+
+[IMPORTANT]
+====
+Currently, {olmv1-first} supports extensions that do not use webhooks and are configured to use the `AllNamespaces` install mode. Extensions that use webhooks or that target a single or specified set of namespaces cannot be installed.
+====
 
 .Prerequisite
 
 * You have added a catalog to your cluster.
-* You have inspected the details of an Operator to find what version you want to install.
+* You have downloaded a local copy of the catalog file.
 
 .Procedure
 
+. Inspect a package for channel and version information from a local copy of your catalog file by running the following commands:
+
+.. Get a list of channels from a selected package by running the following command:
++
+[source,terminal]
+----
+$ jq -s '.[] | select( .schema == "olm.channel" ) | \
+  select( .package == "<package_name>") | \
+  .name' /<path>/<catalog_name>.json
+----
++
+.Example command
+[%collapsible]
+====
+[source,terminal]
+----
+$ jq -s '.[] | select( .schema == "olm.channel" ) | \
+  select( .package == "openshift-pipelines-operator-rh") | \
+  .name' /home/username/rhoc.json
+----
+====
++
+.Example output
+[%collapsible]
+====
+[source,text]
+----
+"latest"
+"pipelines-1.11"
+"pipelines-1.12"
+"pipelines-1.13"
+----
+====
+
+.. Get a list of versions published in a channel by running the following command:
++
+[source,terminal]
+----
+$ jq -s '.[] | select( .package == "<package_name" ) | select( .schema == "olm.channel" ) | select( .name == "<channel_name" ) | .entries | .[] | .name' /<path>/<catalog_name>.json
+----
++
+.Example command
+[%collapsible]
+====
+[source,terminal]
+----
+$ jq -s '.[] | select( .package == "openshift-pipelines-operator-rh" ) | select( .schema == "olm.channel" ) | select( .name == "latest" ) | .entries | .[] | .name' /home/username/rhoc.json
+----
+====
++
+.Example output
+[%collapsible]
+====
+[source,text]
+----
+"openshift-pipelines-operator-rh.v1.11.1"
+"openshift-pipelines-operator-rh.v1.12.0"
+"openshift-pipelines-operator-rh.v1.12.1"
+"openshift-pipelines-operator-rh.v1.12.2"
+"openshift-pipelines-operator-rh.v1.13.0"
+"openshift-pipelines-operator-rh.v1.13.1"
+----
+====
+
 . Create an Operator CR, similar to the following example:
 +
-.Example `test-operator.yaml` CR
+.Example `pipelines-operator.yaml` CR
 [source,yaml]
 ----
 apiVersion: operators.operatorframework.io/v1alpha1
 kind: Operator
 metadata:
-  name: quay-example
+  name: pipelines-operator
 spec:
-  packageName: quay-operator
-  version: 3.8.12
+  packageName: openshift-pipelines-operator-rh
+  channel: <channel>
+  version: <version>
 ----
++
+where:
++
+<channel>:: Optional: Specifies the channel, such as `pipelines-1.11` or `latest` for the package you want to install or update.
+<version>:: Optional: Specifies the specific version or version range, such as `1.11.1`, `1.12.x`, or `>=1.12.1`, of the package you want to install or update. For more information, see "About target versions in OLM 1.0" and "Version range support".
```

Comment:
Note to reviewers: I will add the module for "Version range support" and add the xrefs to the content in a follow up PR.